lib/perl/Genome/Qc/Tool/BamFile.pm

```diff
@@ -18,7 +14,11 @@ sub supports_streaming {
 
 sub cmd_line {
     my $self = shift;
-    return (qw(samtools view), $self->input_file);
+    return (qw(samtools view), $self->gmt_params->{input_file});
```

Comment:
This should probably be concerned with using a particular version via `Genome::Model::Tools::Sam`.